addons/project_mrp/models/project.py

```diff
@@ -70,19 +70,22 @@ def _get_profitability_items(self, with_action=True):
             ['amount'],
             ['account_id'],
         )
-        if mrp_aal_read_group:
+        if not mrp_aal_read_group:
+            mrp_mo_with_aa = self.env['mrp.production'].sudo().search([('analytic_account_id', 'in', self.analytic_account_id.ids)], limit=1)
+        if mrp_aal_read_group or mrp_mo_with_aa:
             can_see_manufactoring_order = with_action and len(self) == 1 and self.user_has_groups('mrp.group_mrp_user')
-            mrp_costs = {
-                'id': mrp_category,
-                'sequence': self._get_profitability_sequence_per_invoice_type()[mrp_category],
-                'billed': sum([res['amount'] for res in mrp_aal_read_group]),
-                'to_bill': 0.0,
-            }
+            mrp_costs = {'id': mrp_category,}
+            if mrp_aal_read_group:
+                mrp_costs.update({
+                    'sequence': self._get_profitability_sequence_per_invoice_type()[mrp_category],
+                    'billed': sum([res['amount'] for res in mrp_aal_read_group]),
+                    'to_bill': 0.0,
+                })
             if can_see_manufactoring_order:
+                costs = profitability_items['costs']
+                costs['data'].append(mrp_costs)
+                costs['total']['billed'] += mrp_costs.get('billed', 0.0)
```

Comment:
I am not a big fan of that "fix".

The project profitability section is more to display the project profitability and not display some buttons by category.

If we don't find any costs/revenues for MO because it is not done then it is expected we don't have the category in that section.

IMO, the customer was expected to see the **stat button** displayed in project update as it was the case in 15 but we remove it in 16, since we don't have the same information with the project profitability, I will ask to Alexandra (AJU) (my PO) her opinion if we reintroduce that stat button or not.

Reply:
I think we have to check with MGM and AJU

Reply:
I think we have to check with MGM and AJU

Reply:
I checked with AJU and she said: "why not including the MO in draft in the compute of project profitability to be able to get that category with the cost/revenues of all MO linked to the project."

Let's see it is possible, I guess yes. 🤷🏼‍♂️

Reply:
Hello @xavierbol can we not just add the MO stat button again at all times? I think that is all we should do as part of this fix. I agree that adding it in the profitability section could be confusing. Then if we want to add the MO total cost in the profitability analysis even when there are no AAL we could think about that for master instead..

Reply:
Hello @mathieu-mgm , indeed you are right, let's add the stat button again. 🙂

Have a nice day! 🙂